packages/near-fast-auth-signer/src/lib/useAuthState.ts

```diff
@@ -59,6 +62,28 @@ export const useAuthState = (skipGetKeys = false): AuthState => {
         } catch {
           setAuthenticated(false);
         }
+      } else {
+        checkFirestoreReady().then(async (isReady) => {
+          if (isReady) {
+            const oidcToken = await firebaseAuth.currentUser.getIdToken();
+            if (window.fastAuthController.getLocalStoreKey(`oidc_keypair_${oidcToken}`)) {
+              const recoveryPK = await window.fastAuthController.getUserCredential(oidcToken);
+              const accountIds = await fetch(`${network.fastAuth.authHelperUrl}/publicKey/${recoveryPK}/accounts`)
```

Comment:
Can we re-use the function?

https://github.com/near/fast-auth-signer/blob/d5c66ff774a73375784f6fdf6c412afb7007cde7/packages/near-fast-auth-signer/src/api/index.ts#L15